writing_content.Rmd

```diff
@@ -490,24 +489,26 @@ We encourage this strategy for several reasons:
     central location and may be helpful for you to re-use the media from
     your course in the future
 -   You can make videos of your slides that can be added to your course
-    using `ari`
+    using [`ari`](https://github.com/jhudsl/ari)
```

Comment:
@howardbaek This might be a good place to talk about loqui? Do you wanna add this?

Reply:
Great idea, I just added a bullet point on Loqui